symbology/symbol-dictionary-sample/src/main/java/com/esri/samples/symbology/SymbolDictionarySample.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2015 Esri. Licensed under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0 Unless required by applicable law
+ * or agreed to in writing, software distributed under the License is
+ * distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the specific language
+ * governing permissions and limitations under the License.
+ */
+
+package com.esri.samples.symbology;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Parent;
+import javafx.scene.Scene;
+import javafx.stage.Stage;
+
+public class SymbolDictionarySample extends Application {
+
+  private static SymbolDictionaryController controller;

```

Comment:
You don't need to set this. This should only be done if you need the controller later. Such as to dispose of the controller's `SceneView`. This sample doesn't have one so you can remove this.